src/Github-api.js

```diff
@@ -0,0 +1,42 @@
+const request = require("request-promise");
+
+const host = 'https://api.github.com';
+
+class Github {
+    constructor() {
+        this.request = request.defaults({
+            json: true,
+            headers: {
+                "Content-Type": "application/json",
+                "User-Agent": "Agent"
+            },
+        });
+    }
+
+    getUser(username) {
+        const path = `${host}/users/${username}`;
+
+        return this.request.get({
+            url: path            
+        });
+    }  
+
+    getUsers() {
```

Comment:
Getting a list of users was not in scope of this HW, but if you would like to implement the "getUsers" route, you would have to user authentication AFAIK + list endpoint doesn't take username as a path parameter, see here: https://docs.github.com/en/rest/users/users#list-users